Standup-Meetings-Getting-Started.md

```diff
@@ -0,0 +1,35 @@
+# Standup Meetings - Getting Started
+
+Stand-up Meeting is one of the two productivity-enhancing features of Probot. To gain maximum benefits, learn how the app works and ways to potentially increase the productivity of your team using this slackbot.
+
+### Create a Slack Account
+To use Probot, a Slack account is a must. All user profiles should be linked to the existing workspace profile on Slack. If your workspace profile is yet to be created, we recommend you to first 'Create Workspace'. Once it is done, create your individual Slack account and add to your workspace.
```

Comment:
"All user profiles should be linked to the existing workspace profile on Slack." we should remove this sentence or we can change it something like "Add all users to slack workspace before using". As this sentence may be confusing to document reader